src/Utilities/MicronTrackerInterface/Cameras.cpp

```diff
@@ -11,89 +11,83 @@
 *
 ***************************************************************/
 
-#include "MTC.h"
-
 #include "Cameras.h"
 #include "MCamera.h"
-//#include "MTVideo.h"
 #include "MicronTrackerLoggerMacros.h"
 
-/****************************/
-/** Constructor */
+//------------------------------------------
 Cameras::Cameras()
 {
-  this->ownedByMe = TRUE;
-  this->mCurrCam = NULL;
-  this->mFailedCam = NULL;
+  this->OwnedByMe = true;
+  this->CurrentCamera = NULL;
+  this->LastFailedCamera = NULL;
   Cameras_HistogramEqualizeImagesSet(true);
   // error handling here
 }
 
-/****************************/
-/** Destructor */
+//------------------------------------------
 Cameras::~Cameras()
 {
   // Clear all previously connected camera
-  for (std::vector<MCamera *>::iterator camsIterator = m_vCameras.begin(); camsIterator != m_vCameras.end(); ++camsIterator)
+  for (std::vector<MCamera*>::iterator camsIterator = CameraList.begin(); camsIterator != CameraList.end(); ++camsIterator)
   {
     free(*camsIterator);
   }
-  m_vCameras.clear();
+  CameraList.clear();

```

Comment:
If you remove m_ prefix then please add this-> to make it clear that it is a member variable.

Reply:
Will do.